README.md

```diff
@@ -6,6 +6,14 @@
 
 Basic repo demoing a simple AVS middleware with full eigenlayer integration. See this [video walkthrough](https://www.loom.com/share/50314b3ec0f34e2ba386d45724602d76?sid=9d68d8cb-d2d5-4123-bd06-776de2076de0).
 
+## Set environment variables
+
+```
+cp .env.template .env
+cp contracts/.env.template contracts/.env
+```
```

Comment:
This file should be copied to contracts/.env in the Makefile, so the user doesn't have to do it manually